src/components/SettingsDialog/Appearance/Appearance.tsx

```diff
@@ -23,34 +35,38 @@ export const Appearance = () => {
-          aria-expanded={showThemeDropdown}
-          onClick={() => setShowThemeDropdown((o) => !o)}
-          onBlur={(e) => handleBlur(e, () => setShowThemeDropdown(false))}
-        >
-          <ThemeSettingsDropdown showDropdown={showThemeDropdown} setShowDropdown={setShowThemeDropdown} />
-        </SettingsButton>
-        {/* <SettingsButton */}
-        {/*   className="appearance-settings_sync-button" */}
-        {/*   onClick={() => { */}
-        {/*     setSyncMode((prevVal) => !prevVal); */}
-        {/*   }} */}
-        {/* > */}
-        {/*   <div className="appearance-settings_sync-button_label"> */}
-        {/*     <p>{t("Appearance.SyncMode")}</p> */}
-        {/*     <p>{t("Appearance.SyncModeDescription")}</p> */}
-        {/*   </div> */}
-        {/*   <SettingsToggle active={syncMode} /> */}
-        {/* </SettingsButton> */}
-        {/* <SettingsButton  */}
-        {/*   className="appearance-settings_notifications-button"  */}
-        {/*   label={t("Appearance.AllowNotifications")} onClick={() => setShowNotifications(!showNotifications)}> */}
-        {/*   <SettingsToggle active={showNotifications} /> */}
-        {/* </SettingsButton> */}
+        <div className="appearance-settings__theme-container">
+          <span className="appearance-settings__theme-title">{t("Appearance.colorScheme")}</span>
+          <div className="appearance-settings__theme-options">
+            <label htmlFor="auto" className="appearence-settings__theme-option" title={t("Appearance.SyncModeDescription")}>
+              <input id="auto" type="radio" value="auto" name="theme" checked={theme === "auto"} onChange={() => setTheme("auto")} />
+              <div className="appearance-settings__auto-preview">
+                <img src={ThemePreviewLight} alt={`${t("Appearance.colorScheme")} Auto`} />
+                <img src={ThemePreviewDark} alt={`${t("Appearance.colorScheme")} Auto`} />
+              </div>
+              <span>
+                <AutoIcon className="appearance-settings__theme-icon" />
+                Auto
+              </span>
+            </label>
+            <label htmlFor="light" className="appearence-settings__theme-option">
+              <input id="light" type="radio" value="light" name="theme" checked={theme === "light"} onChange={() => setTheme("light")} />
+              <img src={ThemePreviewLight} alt={`${t("Appearance.colorScheme")} ${t("Appearance.colorSchemeLight")}`} />
+              <span>
+                <LightIcon className="appearance-settings__theme-icon" />
+                {t("Appearance.colorSchemeLight")}
+              </span>
+            </label>
+            <label htmlFor="dark" className="appearence-settings__theme-option">
+              <input id="dark" type="radio" value="dark" name="theme" checked={theme === "dark"} onChange={() => setTheme("dark")} />
+              <img src={ThemePreviewDark} alt={`${t("Appearance.colorScheme")} ${t("Appearance.colorSchemeDark")}`} />
```

Comment:
Shouldn't have the alt text also `Appearance.colorSchemeDark` as translation instead of only `colorScheme`?

Reply:
It has both, resulting in an alt text of "Color scheme Dark" (or "Farbschema Dunkel" in German)